users/ignoring.rst

```diff
@@ -140,3 +142,42 @@ all files and directories called "foo", ending in a "2" or starting with
   ``some/directory/`` matches the content of the directory, but not the
   directory itself. If you want the pattern to match the directory and its
   content, make sure it does not have a ``/`` at the end of the pattern.
+
+Ignoring files across all devices
+---------------------------------
+
+Although ``.stignore`` file is not synced between the devices, as mentioned
+previously, it's possible to take advantage of an ``#include`` pattern
+to create a single file that will hold shared ignoring patterns.
+
+Assuming the content of a file ``.stignore-shared`` is::
```

Comment:
Assuming the file is called ... and the content of it is ...

Reply:
We should also add `For example` as it's a very detailed and specific example oppose to generic instructions.